Enhanced_Zeus_Modules.sqf

```diff
@@ -14076,6 +14089,42 @@ MAZ_EZM_fnc_initFunction = {
 			["Vehicle repaired.","addItemOk"] call MAZ_EZM_fnc_systemMessage;
 		};
 
+		HYPER_EZM_fnc_setPlateNumber = {
+			params ["_entity"];
+			if (!(_entity isKindOf "Car_F")) exitWith {
+				["This is not a vehicle!","addItemFailed"] call MAZ_EZM_fnc_systemMessage;
+			};
+			[
+				"Set License Plate Number",
+				[ 
+					[
+						"EDIT",
+						"Plate Number (12 chars max)",
+						[ 
+							"", 
+							1 
+						]
+					]
+				], 
+				{
+					params ["_values", "_args", "_display"];
+					private _entity = _args;
+					private _inputText = _values select 0;
+					if (count _inputText > 12) then {
+						_inputText = _inputText select [0, 12];
+					};
+					_entity setPlateNumber _inputText;
```

Comment:
`setPlateNumber` needs to be executed where the `_entity` is local. Needs to be `remoteExec`'d

`[_entity,_inputText] remoteExec ["setPlateNumber"];`